source/common/common/fine_grain_logger.cc

```diff
@@ -85,7 +85,8 @@ std::string FineGrainLogContext::listFineGrainLoggers() ABSL_LOCKS_EXCLUDED(fine
   absl::ReaderMutexLock l(&fine_grain_log_lock_);
   std::string info =
       absl::StrJoin(*fine_grain_log_map_, "\n", [](std::string* out, const auto& log_pair) {
-        absl::StrAppend(out, "  ", log_pair.first, ": ", log_pair.second->level());
+        absl::StrAppend(out, "  ", log_pair.first, ": ",
+                        fmt::to_string(spdlog::level::to_string_view(log_pair.second->level())));
```

Comment:
seems like we should be able to convert a spdlog string_view to an absl::string_view without deep-copy/allocation to a string.

Reply:
thanks, done!